eras/shelley/impl/src/Cardano/Ledger/Shelley/API/Genesis.hs

```diff
@@ -67,6 +68,7 @@ instance
                   (Coin 0)
                   (Coin 0)
                   def
+                  (IStake mempty Map.empty)
```

Comment:
I bet this is what is causing trouble with the benchmarking. the stake in `initialUtxo` needs to be accounted for in the incremental stake.

Reply:
Turned out that was spot-on, thanks a lot @JaredCorduan !

Reply:
:raised_hands: